src/components/dealInDetails/sliderAndTimerComponents/LimitedOffer.tsx

```diff
@@ -38,6 +38,33 @@ const LimitedOffer = () => {
       const { dealSlug } = useParams();
      
 
+
+      const [ days, SetDays] = useState(0);
+      const [hours, setHours] = useState(0)
+      const [minutes, setMinutes] = useState(0);
+      const [seconds, setSeconds] = useState(0);
+    
+      const dates = "2024-03-31T10:59:59.999Z ";
```

Comment:
Why Date is hard coded? when you assign value in variable use `useMemo` hook, other wise it will asing value every then when component rerander.

Reply:
Sir i remove that hardcoded date and assign variable with useMemo hook.